src/graph_compute/chunk_scheduling.h

```diff
@@ -28,6 +28,8 @@ static inline bool interChunkDependency(vid_t v, vid_t w) {
   static const vid_t chunkMask = (1 << CHUNK_BITS) - 1;
   if ((v >> CHUNK_BITS) == (w >> CHUNK_BITS)) {
     return false;
+  } else if ((v & chunkMask) == (w & chunkMask)) {
```

Comment:
`(v & chunkMask) == (w & chunkMask)` is only true if `v` and `w` are in the same chunk, no? Why can this return true if `v` depends on `w` but in the same chunk? It's confusing...